docs/source/man.rst

```diff
@@ -347,7 +350,7 @@ Scans
 
 Use the ``qpc scan`` command to create, run and manage scans.
 
-A scan contains a set of one or more sources of any type plus additional options that refine how the scan runs, such as the products to omit from the scan and the maximum number of parallel system scans. Because a scan can combine sources of different types, you can include network and systems management solution, such as Satellite and vCenter Server, sources in a single scan. When you configure a scan to include multiple sources of different types, for example, a Network source and a Satellite source, the same part of your infrastructure might be scanned more than once. The results for this type of scan could show duplicate information in the reported results. However, you have the option to view the unprocessed detailed report that would show these duplicate results for each source type, or a processed deployments report with deduplicated and merged results.
+A scan contains a set of one or more sources of any type plus additional options that refine how the scan runs, such as the products to omit from the scan and the maximum number of parallel system scans. Because a scan can combine sources of different types, you can include network and systems management solution, such as Openshift, Ansible Automation Platform, Satellite and vCenter Server, sources in a single scan. When you configure a scan to include multiple sources of different types, for example, a Network source and a Satellite source, the same part of your infrastructure might be scanned more than once. The results for this type of scan could show duplicate information in the reported results. However, you have the option to view the unprocessed detailed report that would show these duplicate results for each source type, or a processed deployments report with deduplicated and merged results.
```

Comment:
```suggestion
A scan contains a set of one or more sources of any type, plus additional options that refine how the scan runs, such as the products to omit from the scan, and the maximum number of parallel system scans. Because a scan can combine sources of different types, you can include any combination of Network, OpenShift, Ansible Automation Platform, Satellite, and vCenter Server sources in a single scan. When you configure a scan to include multiple sources of different types, for example a Network source and a Satellite source, the same part of your infrastructure might be scanned more than once. The results for this type of scan could show duplicate information in the reported results. However, you have the option to view the unprocessed detailed report that would show these duplicate results for each source type, or a processed deployments report with deduplicated and merged results.
```